recipes/sompy/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "sompy" %}
+{% set version = "0.1.1" %}
+{% set sha256 = "846626a39e27cca171a81f4388cb30a32cdcd3bedf170c91227cf5a958aaabec" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+sha256: {{ sha256 }}
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - numpy
+
+  run:
+    - python
+    - numpy
```

Comment:
It looks like `matplotlib` is also a `run` requirement. It's not evident when looking at the code up on github, but in the tarball version of the code, it's imported:

https://github.com/ttlg/sompy/commit/1efcb984132424784282f929362421001df1c4a3#diff-8320b10d61a88875c3c04a410d7405caL3